examples/json-csv/json_csv.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/io;
+import ballerina/log;
+
+// Converts a json into an array of strings
+// Returns the result as a tuple which contains the headers and fields
+function getFields(json rec) returns (string[], string[]) {
+    int count = 0;
+    string [] headers = [];
+    string [] fields = [];
+    headers = rec.getKeys();
+    foreach var field in headers {
+        fields[count] = rec[field].toString();
+        count = count + 1;
+    }
+    return (headers, fields);
+}
+
+// Example function which writes json content to csv
+function writeCsv(json content, string path) returns error? {
+    io:WritableCSVChannel csvch = io:openWritableCsvFile(path);
+    int recIndex = 0;
+    int recLen = content.length();
+    while (recIndex < recLen) {
+        (string [], string []) result = getFields(content[recIndex]);
+        var (headers, fields) = result;
+        if (recIndex == 0) {
+            //We ignore the result as this would mean a nill return
+             _  = check csvch.write(headers);
+        }
+        _  = check csvch.write(fields);
+        recIndex = recIndex + 1;
+    }
+    return;
+}
+
+public function main() {
+    // Sample json which will be written
+    json sample = {
+            "employees": {
+                "employee": [
+                    {
+                        "id": "1",
+                        "firstName": "Tom",
+                        "lastName": "Cruise",
+                        "photo": "https://ballerina-team/profile/3737.jpg"
+                    },
+                    {
+                        "id": "2",
+                        "firstName": "Maria",
+                        "lastName": "Sharapova",
+                        "photo": "https://ballerina-team/profile/5676.jpg"
+                    },
+                    {
+                        "id": "3",
+                        "firstName": "James",
+                        "lastName": "Bond",
+                        "photo": "https://ballerina-team/profile/6776.jpg"
+                    }
+                ]
+            }};
+    // Writes json into a csv
```

Comment:
Should this comment be here? This will be highlighted only for the `string path = "./files/sample.csv";` line right?

Reply:
This comment would be applicable for LoC below it.

Reply:
Yeah but on the site it will highlight just the single `string path = ....` line right?